django/core/management/commands/runserver.py

```diff
@@ -181,10 +183,21 @@ def on_bind(self, server_port):
 
         now = datetime.now().strftime("%B %d, %Y - %X")
         version = self.get_version()
+
         print(
             f"{now}\n"
             f"Django version {version}, using settings {settings.SETTINGS_MODULE!r}\n"
             f"Starting development server at {self.protocol}://{addr}:{server_port}/\n"
             f"Quit the server with {quit_command}.",
             file=self.stdout,
         )
+        if not hide_production_warning_message:
+            self.stdout.write(
+                self.style.WARNING(
+                    "WARNING: This is a development server."
+                    " Do not use it in a production deployment. Use a production WSGI "
```

Comment:
```suggestion
                    " Do not use it in a production setting. Use a production WSGI "
```
We are usually saying production setting or production environment in the docs
Note that my suggestions here would require updates to tests